src/graphql/data/TopTokenQuery.ts

```diff
@@ -1,8 +1,37 @@
 import graphql from 'babel-plugin-relay/macro'
-import { TimePeriod, TokenData } from 'hooks/useExplorePageQuery'
 import { useLazyLoadQuery } from 'react-relay'
 
-import type { TopTokenQuery as TopTokenQueryType } from './__generated__/TopTokenQuery.graphql'
+import type { Chain, Currency, TopTokenQuery as TopTokenQueryType } from './__generated__/TopTokenQuery.graphql'
+
+export enum TimePeriod {
+  HOUR,
+  DAY,
+  WEEK,
+  MONTH,
+  YEAR,
+  ALL,
+}
+
+interface IAmount {
+  currency: Currency | null
+  value: number | null
+}
+
+export type TokenData = {
```

Comment:
how could it be null or undefined?

Reply:
that's the type the api returns

Reply:
it returns `string | null | undefined`? what's the diff between when it returns null vs undefined?

Reply:
i dont think any of these will be undefined t'll just be null

Reply:
gives error when don't include undefined for those particular fields that still have it as an option O.o